vocabularies/voc4cat/0007233.ttl

```diff
@@ -8,7 +8,6 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
     skos:definition "In voltammetry: a complementary pair of forward and reverse potential sweeps."@en ;
     skos:historyNote "0009-0008-1278-8890 Michael Götte created the resource"@en ;
     skos:inScheme <https://w3id.org/nfdi4cat/voc4cat_> ;
-    skos:narrower <https://w3id.org/nfdi4cat/voc4cat_0007250> ;
     skos:prefLabel "cycle"@en ;
```

Comment:
At top level such general term should not be so restrictively defined. Would a more focused concept "voltametry cycle" make sense for you?

Reply:
yes makes sense.